docs/docs/configuration/filtering.md

```diff
@@ -394,7 +394,7 @@ $ trivy image --ignorefile ./.trivyignore.yaml python:3.9.16-alpine3.16
 2023-08-31T11:10:27.155+0600	INFO	Vulnerability scanning is enabled
 2023-08-31T11:10:27.155+0600	INFO	Secret scanning is enabled
 2023-08-31T11:10:27.155+0600	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
-2023-08-31T11:10:27.155+0600	INFO	Please see also https://aquasecurity.github.io/trivy/dev/docs/scanner/secret/#recommendation for faster secret detection
+2023-08-31T11:10:27.155+0600	INFO	Please see also https://trivy.dev/dev/docs/scanner/secret/#recommendation for faster secret detection
```

Comment:
This is still `aquasecurity.github.io`:
```
➜  trivy git:(fixlinks) ✗ ./trivy image alpine 
2025-02-11T10:37:40+06:00       INFO    [vuln] Vulnerability scanning is enabled
2025-02-11T10:37:40+06:00       INFO    [secret] Secret scanning is enabled
2025-02-11T10:37:40+06:00       INFO    [secret] If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2025-02-11T10:37:40+06:00       INFO    [secret] Please see also https://aquasecurity.github.io/trivy/dev/docs/scanner/secret#recommendation for faster secret detection

```

Reply:
IIRC you need to update this one:
https://github.com/aquasecurity/trivy/blob/30bcb95350097b322c79e1e16a886a22f3dda42b/pkg/version/doc/doc.go#L20